packages/wifi_info_flutter/wifi_info_flutter/android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.wifi_info_flutter">
```

Comment:
Hi @bparrishMines 
To make the plugin work with Android 8.0 and Android 8.1 without additional steps, you can add that too: `android.permission.CHANGE_WIFI_STATE` .
Thank you